reductstore/src/storage/file_cache.rs

```diff
@@ -12,7 +12,17 @@ use tokio::time::Instant;
 
 use reduct_base::error::ReductError;
 
-pub(crate) type FileRef = Arc<RwLock<File>>;
+pub(super) type FileRef = Arc<RwLock<File>>;
+
+const FILE_CACHE_MAX_SIZE: usize = 1024;
+const FILE_CACHE_TIME_TO_LIVE: Duration = Duration::from_secs(60);
+
+pub(super) fn get_global_file_cache() -> &'static FileCache {
+    static mut FILE_CACHE: OnceLock<FileCache> = OnceLock::new();
+    unsafe {
+        FILE_CACHE.get_or_init(|| FileCache::new(FILE_CACHE_MAX_SIZE, FILE_CACHE_TIME_TO_LIVE))
+    }
+}
```

Comment:
is it at risk? By trying to search for an alternative I saw some people using lazylock: https://doc.rust-lang.org/stable/std/sync/struct.LazyLock.html#